emscripten.py

```diff
@@ -2130,7 +2140,7 @@ def create_sending_wasm(invoke_funcs, jscall_sigs, forwarded_json, metadata):
   if shared.Settings.SAFE_HEAP:
     basic_funcs += ['segfault', 'alignfault']
 
-  basic_vars = ['STACKTOP', 'STACK_MAX', 'DYNAMICTOP_PTR']
+  basic_vars = ['DYNAMICTOP_PTR']
```

Comment:
This seems to remove `STACKTOP` and `STACK_MAX` from the list of globals the JS runtime makes available when instantiating the hosted wasm file. Should the ABI version be incremented?

Reply:
Thanks, bumped the version now.